### PR TITLE
Migrate to eslint 9

### PR DESCRIPTION
**Pull request by Koenkk**
In this PR:
- Migrate to eslint 9
- Swap perfections import sorting for prettier import sorting (via `@ianvs/prettier-plugin-sort-imports`, `trivago/prettier-plugin-sort-imports` throws several `SyntaxError: Unexpected token` errors)

Once merged I will make a similar PR for z2m and zhc

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -5,5 +5,7 @@
     "printWidth": 150,
     "bracketSpacing": false,
     "endOfLine": "lf",
-    "tabWidth": 4
+    "tabWidth": 4,
+    "importOrder": ["", "^node:", "", "<THIRD_PARTY_MODULES>", "", "^zigbee", "", "^[./]"],
```
Comment (Nerivec) [inline .prettierrc:9]: Good call on the alt package.
I'd say let's take advantage of it:
```js
    importOrder: [
        "<TYPES>^(node:)",
        "",
        "<TYPES>",
        "",
        "<TYPES>^[.]",
        "",
        "<BUILTIN_MODULES>",
        "",
        "<THIRD_PARTY_MODULES>",
        "",
        "^zigbee",
        "",
        "^[.]",
    ],
```

Most will never be used, but at least it's nice and tidy.

File: src/utils/backup.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-/* eslint-disable max-len */
-import {fs} from 'mz';
 import * as path from 'path';
 
+import {fs} from 'mz';
```
Comment (Nerivec) [inline src/utils/backup.ts:3]: Should remove this outdated dependency in favor in built-in node equivalent.

File: .prettierrc (hunk 1 above)
Comment (Koenkk) [reply .prettierrc:9]: Done!

File: src/utils/backup.ts (hunk 2 above)
Comment (Koenkk) [reply src/utils/backup.ts]: Done

**Pull request merged**